### ISSUE TITLE
Dragon's Dogma 2 Hangs with Build 955

### ISSUE THREAD

**Issue by siiirk** [praydog/REFramework#1073]
When I initially tested build 955 it worked flawlessly:
[re2_framework_log-workedonce.txt](https://github.com/user-attachments/files/15529030/re2_framework_log-workedonce.txt)

But now when I try to open up the game again, the game stops responding before anything appears in the window (start screen, splash, etc) and has to be terminated.
[re2_framework_log-hang.txt](https://github.com/user-attachments/files/15529034/re2_framework_log-hang.txt)

As before, the problem ceases when dinput9.dll is removed from the game files.

Here are my specs again:
    OS Windows 11
    Graphics card NVIDIA GeForce RTX 4090
    CPU 13th Gen Intel(R) Core(TM) i9-13900HX

And the link to my previous ticket:
[https://github.com/praydog/REFramework/issues/1071](https://github.com/praydog/REFramework/issues/1071)

**Comment by praydog** [comment]
Try the newest nightly build again

**Comment by siiirk** [comment]
Build 958 results in the game automatically closing itself. (Looks like the same behaviour from my previous ticket).
[re2_framework_log.txt](https://github.com/user-attachments/files/15540271/re2_framework_log.txt)

**Comment by siiirk** [comment]
With a little more testing, it looks like I can sometimes (rarely) get the game to start with reframework.
Today I disabled all my mods and verified the game files and managed to get in.
[re2_framework_log-258work.txt](https://github.com/user-attachments/files/15569108/re2_framework_log-258work.txt)
[re2_framework_log-958work2.txt](https://github.com/user-attachments/files/15569109/re2_framework_log-958work2.txt)
I did not load into my save or do anything between the above two tries. I just opened the game, stayed on the main menu, then quit. 
On the second try, I loaded my save and made some minor changes (picked up a seeker's token, used freecam to look around a bit), then saved and quit.
Next attempt, CTD:
[re2_framework_log-958fail.txt](https://github.com/user-attachments/files/15569143/re2_framework_log-958fail.txt)

I haven't been able to get the game to start back up with reframework since. Idk if these logs will help, but I'll drop them here in case. Thanks again for working on this issue 🙏
